src/AutoRest.CSharp/Mgmt/Decorator/CodeModelExtension.cs

```diff
@@ -59,18 +59,46 @@ public static void UpdateSubscriptionIdForAllResource(this CodeModel codeModel)
             }
         }
 
-        public static void UpdateAcronyms(this IEnumerable<Schema> allSchemas)
+        public static void UpdateAcronyms(this CodeModel codeModel)
         {
             if (MgmtContext.MgmtConfiguration.RenameRules.Count == 0)
                 return;
             var transformer = new NameTransformer(MgmtContext.MgmtConfiguration.RenameRules);
             var wordCache = new ConcurrentDictionary<string, string>();
+            // first transform all the name of schemas, properties
+            UpdateAcronyms(codeModel.AllSchemas, transformer, wordCache);
+            // transform all the parameter names
+            UpdateAcronyms(codeModel.OperationGroups, transformer, wordCache);
+        }
+
+        private static void UpdateAcronyms(IEnumerable<Schema> allSchemas, NameTransformer transformer, ConcurrentDictionary<string, string> wordCache)
+        {
             foreach (var schema in allSchemas)
             {
                 TransformSchema(schema, transformer, wordCache);
             }
         }
 
+        private static void UpdateAcronyms(IEnumerable<OperationGroup> operationGroups, NameTransformer transformer, ConcurrentDictionary<string, string> wordCache)
+        {
+            foreach (var operationGroup in operationGroups)
+            {
+                foreach (var operation in operationGroup.Operations)
+                {
+                    TransformOperation(operation, transformer, wordCache);
+                }
+            }
+        }
+
+        private static void TransformOperation(Operation operation, NameTransformer transformer, ConcurrentDictionary<string, string> wordCache)
+        {
+            TransformLanguage(operation.Language, transformer, wordCache);
+            foreach (var parameter in operation.Parameters)
+            {
+                TransformLanguage(parameter.Language, transformer, wordCache);
```

Comment:
I think for a parameter name, we should also keep the camel case. But if I understand correctly, the transformation here might break it. For example, if a parameter has a name like `osProfile`, it will be changed to `OSProfile`, right? .

Reply:
Not really. The code here does change the name of this parameter in the code model from `osProfile` to `OSProfile`. But when we are parsing the code model, [this line of code](https://github.com/Azure/autorest.csharp/blob/c4dd241e65f767b32c6d35eeda93dc3b52f78ff6/src/AutoRest.CSharp/Common/Output/Builders/BuilderHelpers.cs#L72) will be invoked, change the name of the parameter back to `osProfile`.